package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "gql_mini_project",
+  "version": "1.0.0",
+  "main": "app.js",
+  "repository": "https://github.com/ynaydenov-qb/gql_mini_project.git",
+  "author": "Yoan Naydenov <ynaydenov@quickbase.com>",
+  "license": "MIT",
+  "dependencies": {
+    "@graphql-tools/merge": "^9.0.6",
+    "@types/node": "^22.4.1",
+    "apollo-server": "^3.13.0",
+    "dotenv": "^16.4.5",
+    "graphql": "^16.9.0",
+    "ts-node": "^10.9.2",
+    "typescript": "^5.5.4",
+    "uuid": "^10.0.0",
+    "winston": "^3.14.2",
+    "winston-daily-rotate-file": "^5.0.0"
+  },
+  "scripts": {
+    "start": "ts-node src/app.ts",
+    "build": "tsc",
+    "dev": "ts-node-dev src/app.ts"
```

Comment:
`ts-node-dev` is not listed in dependencies. Also, `types` dependencies and `typescript` should be listed in dev dependencies because they're not dependencies used during runtime in production environments. 

https://issuu.com/johan.corrales.teams.test.1/docs/effective_typescript/s/16753105#:~:text=Because%20Type%E2%80%90Script%20is%20a,TypeScript%20generally%20belong%20in%20devDependencies.

Reply:
You are absolutely right, at the beginning I was confused what to add where, thanks for the explanation! I moved the dependencies to dev-dependecies.